providers/base/units/stress/suspend_cycles_reboot.md

```diff
@@ -0,0 +1,96 @@
+<!-- markdownlint-disable MD033 -->
```

Comment:
If possible, try to break lines of text at 80 characters so it's easier to read while in the repository.

Reply:
Hi @pieqq, disable MD033 is used to allow the inline html.